cmd/listener.go

```diff
@@ -76,7 +79,7 @@ var listenCmd = &cobra.Command{
 		restCfg := ctrl.GetConfigOrDie()
 		log, err := setupLogger(defaultCfg.Log.Level)
```

Comment:
Also, I see there is a setupLog and log, and I don't see the differences between it.
If there is no a reason behind it, lets initiate only one logger after config parsing and use it everywhere.